src/evaluation_system/api/parameters.py

```diff
@@ -809,10 +811,105 @@ class MyPlugin(plugin.PluginAbstract):
             )
 
     """
+    base_type = datetime
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    def parse(self, value: str) -> Union[datetime, None]:
+        """
+        Parse a string value into a datetime object.
+
+        Parameters
+        ----------
+        value : str
+            The string value representing a date.
+
+        Returns
+        -------
+        Union[datetime, None]
+            The parsed datetime object if the parsing is successful,
+            otherwise returns None.
+
+        Raises
+        ------
+        ValueError
+            If the value cannot be converted to a datetime object.
+
+        """
+
+        try:
+            if value:
+                return parse(str(value), default=datetime(1978, 1, 1, 0, 0), fuzzy=False)
```

Comment:
Could you please tell me why you chose the default year of 1978? You could choose 1970 or 1 or the current year. I think fuzzy parameters could be nice.

Reply:
I thought that I needed to set a default timestamp, but if it is not necesary I can drop it.

What is the advantage of using fuzzy=True? since I am expecting a string like "date1, date2" I am not leaving many options already.

Reply:
I think with fuzzy things like 2nd of Jul would work. Default is then used to fill up the missing parts of the datetime object. To keep our 2nd of Jul example with your given default value this would lead to:

1978-07-02T00:00 

I hope that's clear.

Reply:
I see. Yes, why not?